core/http.md

```diff
@@ -0,0 +1,3293 @@
+
+{
+  "name": "Blob Created v2",
+  "description": "a cool event",
+  "message": {
+    # Updated definition of a "Blob Created" event excluded for brevity
+  },
+}
+```
+
+```yaml
+HTTP/1.1 200 OK
+Content-Type: application/json; charset=utf-8
+Content-Location: https://example.com/endpoints/ep1/messages/msg1/versions/v2.0
+
+{
+  "messageid": "msg1",
+  "versionid": "1",
+  "self": "https://example.com/endpoints/ep1/messages/msg1/versions/v2.0",
+  "xid": "/endpoints/ep1/messages/msg1/versions/v2.0",
+  "epoch": 2,
+  "name": "Blob Created v2",
+  "isdefault": true,
+  "description": "a cool event",
+  "createdat": "2024-04-30T12:00:00Z",
+  "modifiedat": "2024-04-30T12:00:01Z",
+  "ancestor": "v1.0",
+
+  "message": {
+    # Updated definition of a "Blob Created" event excluded for brevity
+  },
+}
+```
+
+#### `DELETE /<GROUPS>/<GID>/<RESOURCES>/<RID>/versions/<VID>`]
+
+A server MAY support clients deleting a
+[Version](./spec.md#version-entity) of a
+[Resource](./spec.md#resource-entity) via an HTTP `DELETE` directed to the
+Version entity.
+
+The processing of this API is defined in the
+[Deleting Entities](./spec.md#deleting-entities) section of the
+[core specification](./spec.md).
+
+The request MUST be of the form:
+
+```yaml
+DELETE /<GROUPS>/<GID>/<RESOURCES>/<RID>/versions/<VID>
+```
+
+A successful response MUST be of the form:
+
+```yaml
+HTTP/1.1 200 OK
+or
+HTTP/1.1 204 No Content
+```
+
+**Examples:**
+
+Delete a Version:
+
+```yaml
+DELETE /endpoints/ep1/messages/msg1/versions/1.0
+```
+
+```
+HTTP/1.1 204 No Content
+```
+
+Delete a Resource, verifying its `epoch` value:
+
+```yaml
+DELETE /endpoints/ep1/messages/msg1/versions?epoch=5
+```
+
+```
+HTTP/1.1 204 No Content
+```
+
+## Flags / Query Parameters
+
+The [core xRegistry specification](./spec.md) defines a set of
+[flags](./spec.md#flags) that MAY be used to control the processing of requests
+as well as influence how response messages are generated. Each flag is mapped
+to an HTTP query parameter, as defined in the following sections:
+
+- [`?binary` Flag](#binary-flag)
+- [`?collections` Flag](#collections-flag)
+- [`?doc` Flag](#doc-flag)
+- [`?epoch` Flag](#epoch-flag)
+- [`?filter` Flag](#filter-flag)
+- [`?ignoredefaultversionid` Flag](#ignoredefaultversionid-flag)
+- [`?ignoredefaultversionsticky` Flag](#ignoredefaultversionsticky-flag)
+- [`?ignoreeopch` Flag](#ignoreepoch-flag)
+- [`?ignorereadonly` Flag](#ignorereadonly-flag)
+- [`?inline` Flag](#inline-flag)
+- [`?setdefaultversionid` Flag](#setdefaultversionid-flag)
+- [`?sort` Flag](#sort-flag)
+- [`?specversion` Flag](#specversion-flag)
+
+### `?binary` Flag
+
+A server MAY support the `?binary` flag/query parameter allowing clients to
+request that any Resources returned in the response always have its
+domain-specific documents be serialized under its `<RESOURCE>base64`
+ attribute.
+
+See [Binary Flag](./spec.md#binary-flag) for more information.
+
+This query parameter MUST be serialize as:
```

Comment:
If we keep this section, "serialize" => "serialized" in all cases.

Reply:
done